core/math/big/internal.odin

```diff
@@ -2047,7 +2047,8 @@ internal_int_inverse_modulo :: proc(dest, a, b: ^Int, allocator := context.alloc
 
 	return _private_inverse_modulo(dest, a, b)
 }
-internal_invmod :: proc{ internal_int_inverse_modulo, }
+internal_int_invmod :: internal_int_inverse_modulo
+internal_invmod :: proc{ internal_int_invmod, }
```

Comment:
Why change the proc group? It can remain `internal_invmod :: proc{ internal_int_inverse_modulo, }`.

Reply:
I was mimicking how you have for example internal_addmod point to internal_int_ addmod process group etc. But I can certainly change that.